lib/clearsale/object.rb

```diff
@@ -1,5 +1,4 @@
 require 'json'
-require 'ostruct'
 
 module Clearsale
```

Comment:
Not a fan of this name since it is a primary ruby class. I think this should be instead `Response`